DevAcademy/Scenes/Places/PlacesDetail.swift

```diff
@@ -24,22 +24,18 @@ struct PlacesDetail: View {
                     ProgressView()
                 }
             VStack(spacing: 200){
-                    Text(feature.properties.nazev)
-                        .font(.title2)
-                        .fontWeight(.semibold)
-                        .lineLimit(1)
-                    Text(feature.properties.druh.rawValue)
-                        .foregroundColor(.secondary)
-                        .font(.subheadline)
-                        
-                }
+                Text(state.placeTitle)
+                    .font(.title2)
+                    .fontWeight(.semibold)
+                    .lineLimit(1)
+            }
            
             }
         }
 }
 
-struct PlacesDetail_Previews: PreviewProvider {
-    static var previews: some View {
-        PlacesDetail(feature: Features.mock.features[0])
-    }
-}
+//struct PlacesDetail_Previews: PreviewProvider {
+//    static var previews: some View {
+//        PlacesDetail(feature: Features.mock.features[0])
```

Comment:
Kdyz upravis ten init na ten novej s tim state (`PlacesDetail(state: PlacesDetailStateView(feature: .mock))`) tak to muzes klidne odkomentovat a zase pouzivat. 
Nebo to klidne cely smazni, jestli previews nechces pouzivat. 😉